utils/searx.sh

```diff
@@ -88,7 +88,7 @@ BUILD_PACKAGES_centos="\
 firefox graphviz graphviz-gd ImageMagick librsvg2-tools
 texlive-xetex-bin texlive-collection-fontsrecommended
 texlive-collection-latex dejavu-sans-fonts dejavu-serif-fonts
-dejavu-sans-mono-fonts"
+dejavu-sans-mono-fonts fontforge ttfautohint"
```

Comment:
I haven't tested centos-7 (yum package manager), I don't know if this works .. see https://github.com/searxng/searxng/pull/151#discussion_r653031106

Lets ignore CentOS-7 as developer (build) platform.